### PR TITLE
Cleanup of ipvs utils

### PR DESCRIPTION
**Pull request by rramkumar1**
**What this PR does / why we need it**:
This PR is a small cleanup of pkg/util/ipvs.

The changes are as follows:

1. Rename toBackendService -> toIPVSService and toBackendDestination -> toIPVSDestination. The use of the term 'backend' makes things really confusing and this renamig makes it more explicit about what we are doing.

2. Give the libnetwork/ipvs package the name libipvs. This makes it less confusing since the package these files are in is also ipvs.

3. Some variable naming cleanup to make things easier to read.

/assign @m1093782566 

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/util/ipvs/ipvs.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type Interface interface {
 	DeleteRealServer(*VirtualServer, *RealServer) error
 }
 
-// VirtualServer is an user-oriented definition of an IPVS virtual server in its entirety.
+// VirtualServer is an user-oriented definition of an IPVS Server in its entirety.
```
Comment (m1093782566) [inline pkg/util/ipvs/ipvs.go]: Tere are two "server" in IPVS: virtual server and real server. So, IPVS Server will make people wonder what the "server" is?

File: pkg/util/ipvs/ipvs.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ func (svc *VirtualServer) String() string {
 	return net.JoinHostPort(svc.Address.String(), strconv.Itoa(int(svc.Port))) + "/" + svc.Protocol
 }
 
-// RealServer is an user-oriented definition of an IPVS real server in its entirety.
+// RealServer is an user-oriented definition of an IPVS Destination in its entirety.
```
Comment (m1093782566) [inline pkg/util/ipvs/ipvs.go]: I discussed with @thockin before and there is a question about "real server" vs. "destination", which one is better?

And, we decided to follow the concept of IPVS.

File: pkg/util/ipvs/ipvs.go (hunk 1 above)
Comment (rramkumar1) [reply pkg/util/ipvs/ipvs.go]: Makes sense, changed back.

File: pkg/util/ipvs/ipvs.go (hunk 2 above)
Comment (rramkumar1) [reply pkg/util/ipvs/ipvs.go]: Makes sense, changed back.

**Comment by m1093782566** [comment]
/lgtm

/approve

Thanks for cleaning up!

**Pull request merged**